html-skeleton/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Skeletons</title>
+    <body>
+      Hello, HTML!
+    </body>
+  </head>
```

Comment:
Hi Matt, try moving the `<body>` element below the `<head>` element. They should be siblings 

```html
<head>
  <title></title>
</head>
<body>
</body>
```